arch/arm/src/lc823450/lc823450_dvfs2.c

```diff
@@ -127,7 +127,7 @@ static uint64_t _get_current_time64(void)
 {
   struct timespec ts;
 
-  clock_gettime(CLOCK_MONOTONIC, &ts);
+  clock_systime_timespec(&ts);
```

Comment:
Not related to this change, but just noticed that function is named `_get_current_time64` and `up_get_current_time` is specified few lines above in the description

Reply:
Done.